### ISSUE TITLE
libkrb5 1.18 missing krb5_rc_resolve_full krb5_rc_close

### ISSUE THREAD

**Issue by commonism** [hautreux/auks#60]
Building the the [scibian auks package](https://github.com/scibian/auks) for debian bullseye runs into

```
/usr/bin/ld: ../api/auks/.libs/libauksapi.so: undefined reference to `krb5_rc_close'
/usr/bin/ld: ../api/auks/.libs/libauksapi.so: undefined reference to `krb5_rc_resolve_full'
```
the symbols are missing from libkrb5, they were renamed to k5_rc_close & k5_rc_resolve https://github.com/krb5/krb5/commit/dcb853ac32779b173f39e19c0f24b0087de85771.

This is the code in question:
https://github.com/hautreux/auks/blob/b81c595ef73661a54924fb71dae3ce68d1be03a8/src/api/auks/auks_krb5_stream.c#L874-L914

According to [gentoo#738968 www-apache/mod_auth_kerb-5.4-r2 references krb5_rc_resolve_full, but it is removed from libkrb5.so](https://bugs.gentoo.org/738968)
> RCACHE is mandatory these days so hardcode

Therefore I assume the code in question can be removed (as with the gentoo patch to mod_auth_kerb).
Is this correct?

**Comment by rezib** [comment]
Note that PR #53 fixes this.

**Comment by hautreux** [comment]
fixed in 14ef6c9e2e2b73dfac9d612eb23f652dcfd231 (upgrade to version 0.5.3 to get it)
